src/Resources/views/storefront/page/checkout/finish/index.html.twig

```diff
@@ -0,0 +1,26 @@
+{% sw_extends '@Storefront/storefront/page/checkout/finish/index.html.twig' %}
+
+{% block page_checkout_finish_details %}
+    {{ parent() }}
+    {% if context.shippingMethod.name == "InPost" %}
```

Comment:
Dont use shipping method name. We have custom field for that, because everyone can change it. 
context.shippingMethod.customFields.technical_name - something like this, but You need to check it.
Use const to check technicalName - constant('BitBag\\ShopwareInPostPlugin\\Factory\\ShippingMethodPayloadFactoryInterface::SHIPPING_KEY')

Reply:
Fixed in next commit